docs/Configuring-Orion/Configuration-File.md

```diff
@@ -22,7 +22,7 @@ Specify the configuration file when [starting Orion](../Reference/Orion-CLI-Synt
 | `libsodiumpath`          | Optional | Path to libsodium shared library                                                          | [Dependant on OS](#libsodiumpath) |
 | `alwayssendto`           | Optional | List of files containing public keys to include as recipients for every transaction       | `[]`                              |
 | `passwords`              | Optional | File containing [passwords](#passwords) to unlock `privatekeys`                           | `Not set`                         |
-| `storage`                | Optional | [Storage](#storage) for payloads and related information. To use PostgreSQL, see [Using PostgreSQL](Using-PostgreSQL.md).                                | `"leveldb"`                       |
+| `storage`                | Optional | [Storage](#storage) for payloads and related information. To use PostgreSQL, see [Using PostgreSQL](Using-PostgreSQL.md). To use Oracle, see [Using Oracle](Using-Oracle.md).                               | `"leveldb"`                       |
```

Comment:
I think you could combine the two references: 
Relational databases [PostgreSQL] and [Oracle] are supported. 

We generally try and avoid references of the type 'For more information on, see x' in favour of linking the topic in the sentence.  It's not always avoidable but where it's possible, it generally reduces wordiness.